src/site/_includes/components/Codepen.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {html} = require('common-tags');
+const {escape, stringify} = require('querystring');
+const iframe = require('./IFrame');
+
+/** @type {string} */
+const THEME_ID = '38982';
+
+/** @type {string} */
+const DEFAULT_TAB = 'result';
```

Comment:
I think we should make this user configurable.

Reply:
Done